src/Authorization.AspNetCore/AuthorizationError.cs

```diff
@@ -16,8 +13,8 @@ public class AuthorizationError : ValidationError
         /// <summary>
         /// Initializes a new instance of the <see cref="AuthorizationError"/> class for a specified authorization result.
         /// </summary>
-        public AuthorizationError(INode node, ValidationContext context, OperationType? operationType, AuthorizationResult result)
-            : this(node, context, GenerateMessage(operationType, result), result)
+        public AuthorizationError(INode node, ValidationContext context, OperationType? operationType, string message, AuthorizationResult result)
```

Comment:
Let's merge two ctors into one, move `OperationType? operationType` at the end: `OperationType? operationType = null`.